src/EFCore.PG/Query/ExpressionTranslators/Internal/NpgsqlFullTextSearchMethodTranslator.cs

```diff
@@ -34,47 +37,45 @@
 
 namespace Npgsql.EntityFrameworkCore.PostgreSQL.Query.ExpressionTranslators.Internal
 {
+    /// <summary>
+    /// Provides translations for PostgreSQL full-text search methods.
+    /// </summary>
     public class NpgsqlFullTextSearchMethodTranslator : IMethodCallTranslator
     {
-        static readonly MethodInfo _tsQueryParse = typeof(NpgsqlTsQuery).GetMethod(
-            nameof(NpgsqlTsQuery.Parse),
-            BindingFlags.Public | BindingFlags.Static);
+        static readonly MethodInfo TsQueryParse =
+            typeof(NpgsqlTsQuery).GetMethod(nameof(NpgsqlTsQuery.Parse), BindingFlags.Public | BindingFlags.Static);
 
-        static readonly MethodInfo _tsVectorParse = typeof(NpgsqlTsVector).GetMethod(
-            nameof(NpgsqlTsVector.Parse),
-            BindingFlags.Public | BindingFlags.Static);
+        static readonly MethodInfo TsVectorParse =
+            typeof(NpgsqlTsVector).GetMethod(nameof(NpgsqlTsVector.Parse), BindingFlags.Public | BindingFlags.Static);
 
-        static readonly IReadOnlyDictionary<string, string> _sqlNameByMethodName =
-            new Dictionary<string, string>
-            {
-                [nameof(NpgsqlFullTextSearchDbFunctionsExtensions.ArrayToTsVector)] = "array_to_tsvector",
-                [nameof(NpgsqlFullTextSearchDbFunctionsExtensions.ToTsVector)] = "to_tsvector",
-                [nameof(NpgsqlFullTextSearchDbFunctionsExtensions.PlainToTsQuery)] = "plainto_tsquery",
-                [nameof(NpgsqlFullTextSearchDbFunctionsExtensions.PhraseToTsQuery)] = "phraseto_tsquery",
-                [nameof(NpgsqlFullTextSearchDbFunctionsExtensions.ToTsQuery)] = "to_tsquery"
-            };
-
-        public Expression Translate(MethodCallExpression methodCallExpression)
+        static readonly IReadOnlyDictionary<string, string> SQLNameByMethodName = new Dictionary<string, string>
```

Comment:
I think `Sql` is better than `SQL`, this is how EF Core working does it in general (e.g. `MigrationsSqlGenerator`)